src/ui.rs

```diff
@@ -3048,40 +3048,60 @@ impl Component for SectionLineView<'_> {
     }
 
     fn draw(&self, viewport: &mut Viewport<Self::Id>, x: isize, y: isize) {
-        const NEWLINE_ICON: &str = "⏎";
-        let Self { line_key: _, inner } = self;
+        fn replacement_characters(character: char) -> Option<&'static str> {
```

Comment:
```suggestion
        fn replace_control_character(character: char) -> Option<&'static str> {
```

suggestion: make function name into a verb